superflore/generators/bitbake/ros_meta.py

```diff
@@ -31,24 +31,51 @@ def __init__(
 
     def clean_ros_recipe_dirs(self, distro=None):
         if distro:
-            info('Cleaning up recipes-ros-{0} directory...'.format(distro))
-            self.repo.git.rm('-rf', 'recipes-ros-{0}'.format(distro))
+            info(
+                'Cleaning up generated-recipes-{} directory...'.format(distro))
+            self.repo.git.rm('-rf', 'generated-recipes-{}'.format(distro))
         else:
-            info('Cleaning up recipes-ros-* directories...')
-            self.repo.git.rm('-rf', 'recipes-ros-*')
+            info('Cleaning up generated-recipes-* directories...')
+            self.repo.git.rm('-rf', 'generated-recipes-*')
 
     def commit_changes(self, distro):
         info('Adding changes...')
         if distro == 'all':
             commit_msg = 'regenerate all distros, {0}'
-            self.repo.git.add('recipes-ros-*')
+            self.repo.git.add('generated-recipes-*')
```

Comment:
I'm not sure I want to call the folder `generated-recipes`. Is there any reason to not use `recipes-ros`?

Reply:
The repository will contain both auto-generated files (by SuperFlore) and manually managed files (overrides when/if needed). ```generated-recipes``` folder indicates that all content was auto-generated by SuperFlore.

Please see from [wiki](https://github.com/ros/meta-ros/wiki/Proposed-Superflore-OE-Recipe-Generation-Scheme):
````
meta-ros will only support ROS distros for which there are generated recipes. Manually created recipes under recipes-{ros1,ros2}for which there is not a generated equivalent will be moved to unused-recipes-{ros1,ros2}. Manually created recipes for which there is a generated equivalent will initially be moved to v1-recipes-{ros1,ros2} and eventually removed.
```

Reply:
Alright, sounds good.